API/Queries/RootQuery.cs

```diff
@@ -24,7 +27,7 @@ public RootQuery(IDataRetrievalService dataRetrievalService)
                     }
                 },
                 context =>
-                    dataRetrievalService.GetForecast(
+                    metAPIService?.GetCompactForecast(
```

Comment:
How come metAPIService is nullable?

Reply:
Because we resolve the interface from the service provider (check out a few lines above this). If we've configured our services correctly this shouldn't occur, but it's theoretically possible hence the nullability.

Reply:
Ah, of course. Thanks! :)